projects/frontend/src/components/admin-panel/AdminPanel.tsx

```diff
@@ -0,0 +1,178 @@
+import React, { FC, useCallback, useState, useEffect } from 'react';
+import { SWRResponse } from 'swr';
+import styled from 'styled-components';
+import { CSSTransition, TransitionStatus } from 'react-transition-group';
+import Sound from '../../models/sound';
+import SearchContainer from '../features/SearchContainer';
+import PanelSound from './PanelSound';
+import PanelInfoContainer from './PanelInfoContainer';
+import Notification from './Notification';
+import { textShadowVisibility } from '../../styles/mixins';
+
+interface AdminStyleProps {
+  state?: TransitionStatus;
+}
+
+const AdminPanelMain = styled.div<AdminStyleProps>`
+  position: absolute;
+  transition: bottom 0.4s ease-out;
+  bottom: ${ props => props.state === 'entered' || props.state === 'entering' ? '0px' : '-110vh' };
+  transition: top 0.4s ease-out;
+  top: ${ props => props.state === 'entered' || props.state === 'entering' ? '104px' : '100vh' };
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  overflow-y: hidden;
+  z-index: 100;
+  background-color: ${ props => props.theme.colors.bg };
+`;
+
+const AdminFeatures = styled.div`
+  padding: 12px 30px 20px 30px;
+  box-shadow: 0px 5px 5px 2px ${ props => props.theme.colors.shadowDefault };
+`;
+
+const FeaturesHeader = styled.div`
+  display: flex;
+  align-items: left;
+
+  > h2 {
+    ${ textShadowVisibility }
+    margin: 18px 20px 10px 0px;
+
+    &:first-child {
+      color: ${ props => props.theme.colors.borderDefault };
+    }
+  }
+`;
+
+const CloseBar = styled.div`
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  opacity: 0.5;
+  background-color: ${ props => props.theme.colors.borderRed };
+  border-radius: 8px;
+  padding: 5px 0px;
+  cursor: pointer;
+  box-shadow: 0px 3px 4px ${ props => props.theme.colors.shadowDefault };
+
+  &:hover {
+    opacity: 0.7;
+  }
+
+  > p {
+    color: black;
+    font-weight: bold;
+    opacity: 0.8;
+    margin: 0;
+  }
+`;
+
+const LowerContainer = styled.div`
+  display: flex;
+  overflow-y: hidden;
+  flex-grow: 1;
+
+  @media only screen and (max-width: 780px) {
+    flex-direction: column;
+  }
+`;
+
+const SoundsContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  overflow-y: scroll;
+  padding: 20px 30px;
+  border-right: 5px solid ${ props => props.theme.colors.borderDefault };
+
+  > h2 {
+    color: ${ props => props.theme.colors.borderDefault };
+    margin: 5px 0px;
+    
+    ${ textShadowVisibility }
+  }
+
+  &::-webkit-scrollbar {
+    width: 15px;
+    background: ${ props => props.theme.colors.innerB };
+  }
+
+  &::-webkit-scrollbar-thumb {
+    background: ${ props => props.theme.colors.innerA };
+  }
+
+  @media only screen and (max-width: 780px) {
+    border-right: none;
+    border-bottom: 5px solid ${ props => props.theme.colors.borderDefault };
+  }
+`;
+
+interface AdminPanelProps {
+  showAdminPanel: boolean;
```

Comment:
```suggestion
  open: boolean;
```

`show` is also acceptable, but there's no need to say `AdminPanel` since this is part of `AdminPanelProps`